core/src/main/scala/sttp/openai/Main.scala

```diff
@@ -6,8 +6,9 @@ object Main extends App {
   val backend: SyncBackend = DefaultSyncBackend()
 
   val openAi: OpenAi = new OpenAi("sk-")
```

Comment:
Let's maybe not modify the main, idealiy would be to clear it or leave it as it was?